net.vdrift.VDrift.yaml

```diff
@@ -0,0 +1,87 @@
+command: vdrift
+rename-icon: vdrift
+rename-desktop-file: vdrift.desktop
+rename-appdata-file: vdrift.appdata.xml
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --socket=pulseaudio
+
+modules:
+  - name: scons
+    buildsystem: simple
+    cleanup:
+      - "*"
+    sources:
+      - type: archive
+        url: http://downloads.sourceforge.net/project/scons/scons/3.0.4/scons-3.0.4.tar.gz
+        sha256: e2b8b36e25492720a05c0f0a92a219b42d11ce0c51e3397a1e8296dfea1d9b1a
+    build-commands:
+      - python setup.py install --prefix=${FLATPAK_DEST} --optimize=1
+
+  - name: bullet
+    buildsystem: cmake
+    config-opts:
+      - -DBUILD_BULLET2_DEMOS=OFF
+      - -DBUILD_BULLET3=OFF
+      - -DBUILD_CPU_DEMOS=OFF
+      - -DBUILD_EXTRAS=OFF
+      - -DBUILD_OPENGL3_DEMOS=OFF
+      - -DBUILD_UNIT_TESTS=OFF
+      - -DUSE_GLUT=OFF
+      - -DUSE_GRAPHICAL_BENCHMARK=OFF
+      - -DUSE_DOUBLE_PRECISION=ON
+      - -DBULLET2_MULTITHREADING=ON
+    sources:
+      - type: git
+        url: https://github.com/bulletphysics/bullet3.git
+        tag: 3.25
+        commit: 2c204c49e56ed15ec5fcfa71d199ab6d6570b3f5
+
+  - name: vdrift
+    buildsystem: simple
+    build-commands:
+      - scons revision="exported" prefix=${FLATPAK_DEST} release=1 verbose=1 minimal=0 use_binreloc=0  NLS=1 datadir=share/vdrift/data localedir=share/locale
+      - scons install prefix=${FLATPAK_DEST} bindir=bin
+    post-install:
+      - install -Dm644 data/textures/icons/vdrift-16x16.png ${FLATPAK_DEST}/share/icons/hicolor/16x16/apps/vdrift.png
+      - install -Dm644 data/textures/icons/vdrift-32x32.png ${FLATPAK_DEST}/share/icons/hicolor/32x32/apps/vdrift.png
+      - install -Dm644 data/textures/icons/vdrift-64x64.png ${FLATPAK_DEST}/share/icons/hicolor/64x64/apps/vdrift.png
+      - install -Dm644 vdrift-128x128.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/vdrift.png
+      - install -Dm644 vdrift.appdata.xml ${FLATPAK_DEST}/share/appdata/vdrift.appdata.xml
+      - install -Dm644 vdrift.desktop ${FLATPAK_DEST}/share/applications/vdrift.desktop
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/vdrift/vdrift/vdrift-2014-10-20/vdrift-2014-10-20.tar.bz2
+        sha256: 458d45d59075c2ce943ec6bc271a5bd0eb6bc6ed123670b65038703345020be3
+      - type: file
+        path: vdrift-128x128.png
+      - type: patch
+        path: scons3.patch
+      - type: patch
+        path: scons3in.patch
+      - type: patch
+        path: bullet-double.patch
+      - type: patch
+        path: gcc6.patch
+      - type: patch
+        path: appdata1.patch
+        use-git: true
+        options:
+          - --ignore-whitespace
+      - type: patch
+        path: appdata2.patch
+        use-git: true
+        options:
+          - --ignore-whitespace
+      - type: patch
+        path: appdata3.patch
+        use-git: true
+        options:
+          - --ignore-whitespace
```

Comment:
```suggestion
      - type: patch
        paths:
          - appdata1.patch
          - appdata2.patch
          - appdata3.patch
        use-git: true
        options:
          - --ignore-whitespace
```

Reply:
I tried that. Does not work.

Reply:
define "does not work". I use that form quite often.